Sources/Target/ProcessBase.cpp

```diff
@@ -184,13 +184,14 @@ ErrorCode ProcessBase::beforeResume() {
   if (!isAlive())
     return kErrorProcessNotFound;
 
-  SoftwareBreakpointManager *bpm = softwareBreakpointManager();
-
   //
   // Enable breakpoints.
   //
-  if (bpm != nullptr) {
-    bpm->enable();
+  for (auto bpm : std::initializer_list<BreakpointManager *>{
```

Comment:
You should use `std::list` here, it's functionally equivalent and is shorter to type 😆